app/models/comment.rb

```diff
@@ -56,11 +56,11 @@ def text= text
   end
 
   def people_allowed_to_be_mentioned
-    @people_allowed_to_be_mentioned ||= if parent.public?
-                                          "all"
-                                        else
-                                          [*parent.participants, parent.author].map(&:id).uniq
-                                        end
+    if parent.public?
+      "all"
+    else
+      [*parent.participants, parent.author].map(&:id).uniq
```

Comment:
While thinking about the filter in the notifications, I remembered, that this doesn't work as expected. With the new federation, `participations` only contain participations of all participants, if the owner of the post is local, otherwise it only contains the local participations.

While this is perfect for the notification filter (means less person_ids in the `IN` query, because it only contains local IDs), this doesn't work for the send filter. Do we really want all participants to be mentionable (comment, like or poll_participation?), or only people who commented?

I suggest the latter, so it would be: `[*parent.comments.pluck(:author_id), parent.author_id].uniq`